woodwork/tests/test_utils.py

```diff
@@ -173,6 +173,16 @@ def test_read_file_errors_no_content_type(sample_df_pandas, tmpdir):
         ww.read_file(filepath=filepath)
 
 
+def test_read_file_errors_unsupported(sample_df_pandas, tmpdir):
+    filepath = os.path.join(tmpdir, 'sample')
+    sample_df_pandas.to_feather(filepath)
+
+    content_type = "application/feather"
+    not_supported_error = "Reading from content type {} is not currently supported".format(content_type)
+    with pytest.raises(RuntimeError, match=not_supported_error):
+        ww.read_file(filepath=filepath, content_type=content_type)
+
+
 def test_read_file_errors_unhandled_content_type(sample_df_pandas, tmpdir):
```

Comment:
Isn't this test `test_read_file_errors_unhandled_content_type` still identical to `test_read_file_errors_no_content_type` or am I missing something?

Reply:
Yep, removed the duplicate test.